### PR TITLE
[skip ci] WIP-Implement OVA installer tests

### LINKED ISSUE #5406: Implement OVA installer tests
**User Statement:**
As a developer I want to validate that services on the OVA start.

**Details:**
Deploy and test the OVA after it builds

**Acceptance Criteria:**

- [ ] Set up Robot for `vic-product` repo
- [ ] Deploy the OVA to a vCenter (no VC on CI yet, so the test won't be enabled on CI)
- [ ] Get the IP of the OVA and verify that web services are running (`admiral`, `harbor`, `fileserver`, `engine_installer`)

### PR DESCRIPTION
**Pull request by singhshwetaQE**
Fixes #5406 

This test may need little more finishing touch, was working on my local machine.

### CODE REVIEW

File: tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+GOVC Use ESXi Environment
+        Set Environment Variable  GOVC_URL  @{vm-ips}[0]
+        Set Environment Variable  GOVC_USERNAME  root
+        Set Environment Variable  GOVC_PASSWORD
+
+*** Test Cases ***
+Deploy VIC-OVA To Test Server
+  [Arguments]  ${dhcp}=True  ${protocol}=http  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL_ARRAY}  ${datastore}=%{TEST_DATASTORE}  ${cluster}=%{TEST_RESOURCE}  ${datacenter}=%{TEST_DATACENTER}
+  Set Global Variable  ${ova_path}  bin/vic-v1.2.0-d0ea01c2.ova
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: This should be set by environment variable

File: tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  Log To Console  \nStarting to deploy unified-ova to test server...
+  ${out}=  Run Keyword If  ${dhcp}  Run  ovftool --datastore=${datastore} ${ova_options} --net:"Network"="vm-network" ${ova_path} 'vi://Administrator@vSphere.local:Admin\!23@${host}/${datacenter}/host/${cluster}'
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: Username and password should be set by environment variable

File: tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    Set Suite Variable  ${vm-names}  ${vm-names}
+    Set Suite Variable  ${vm-ips}  ${vm-ips}
+    GOVC Use VC Environment
+
+Cleanup Suite Environment
+        Nimbus Cleanup  ${vm-names}  ${not_collect_log}
+
+GOVC Use VC Environment
+        Set Environment Variable  GOVC_URL  ${vc-ip}
+        Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: Username and password should be set by environment variable or consistent with the other manual tests

File: tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  \   Exit For Loop If  ${status}
+
+   Set Environment Variable  Appliance-IP  ${ip}
+   Sleep  30s
+
+   Log To Console  Waiting for Getting Started Page to Come Up...
+   :FOR  ${i}  IN RANGE  20
+    \  ${rc}=  Run And Return Rc  curl -sk https://%{Appliance-IP}:9443 -XPOST -F target=${host} -F user=Administrator@vsphere.local -F password=Admin!23
+    \  Should Be Equal As Integers  ${rc}  0
+
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: Do the services come up here without hitting the register API? https://github.com/vmware/vic-product/pull/456

File: tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  \   ${ip}=  Run Keyword If  ${status}  Fetch From Right  ${line}  ${SPACE}
+  \   Log To Console  ${ip}
+  \   Exit For Loop If  ${status}
+
+   Set Environment Variable  Appliance-IP  ${ip}
+   Sleep  30s
+
+   Log To Console  Waiting for Getting Started Page to Come Up...
+   :FOR  ${i}  IN RANGE  20
+    \  ${rc}=  Run And Return Rc  curl -sk https://%{Appliance-IP}:9443 -XPOST -F target=${host} -F user=Administrator@vsphere.local -F password=Admin!23
```
Comment (andrewtchin) [inline tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: Username and password should use variables

Comment (andrewtchin) [reply tests/manual-test-cases/Group19-VIC-Product/19-11-OVA-installer.robot]: If this is hitting the register API shouldn't it be `/register`? https://github.com/vmware/vic-product/pull/456

**Comment by singhshwetaQE** [comment]
Manjunath has incorporated this changes into new test

**Comment by singhshwetaQE** [comment]
Manjunath has incorporated this changes into a new test , so this will be duplicate PR- Closing it

**Pull request closed without merging**